src/first_steps/commandline_rizin.md

```diff
@@ -12,7 +12,7 @@ The standard format for Rizin commands is as follows:
 # executes a rizin command at at the current address (0x00000000)
 [0x00000000]> command <params>
 # executes a rizin command with a special temporary modifier
-[0x00000000]> command <params> @<temporay modifier>
+[0x00000000]> command <params> @<temporary modifier>
```

Comment:
maybe we should call `params` as `args` ?

Reply:
ok, it's more accurate 5658011614da5b5bf03e8ebbffc86e7fa47b209a